Cogs/CogManagement.py

```diff
@@ -24,15 +26,14 @@ async def load(self, interaction:discord.Interaction, cog_name:str):
 
         Args:
             cog_name (str): Name of the cog being loaded
+                cog_name will be autofilled with all of the cogs we currently have
 
         Outputs:
             Message to user informing them of what cog is being loaded, and when the action is done.
         """
 
-        # Finds the absolute path to the cog that will be loaded
         file = abspath('Cogs/' + cog_name + '.py')
```

Comment:
Re-add this comment and line 35's comment to keep consistent comments along the functions please.